launchlib/launcher.go

```diff
@@ -311,13 +312,17 @@ func filterHeapSizeArgs(args []string) []string {
 	}
 
 	if !hasInitialRAMPercentage && !hasMaxRAMPercentage {
-		filtered = append(filtered, "-XX:InitialRAMPercentage=75.0")
-		filtered = append(filtered, "-XX:MaxRAMPercentage=75.0")
+		initialHeapPercentage, err := computeInitialHeapPercentage(customConfig)
+		if err != nil {
+			initialHeapPercentage = 75.0
+		}
+		filtered = append(filtered, fmt.Sprintf("-XX:InitialRAMPercentage=%.1f", initialHeapPercentage))
```

Comment:
I don't think additional precision beyond 1 decimal place will have much of an effect, but am open to changing this.

Reply:
Instead of computing specific percentage, let's set `-Xmx` and `-Xms` based on the desired heap size, since those args are better known/understood than the `Initial`/`MaxRAMPercentage`

Reply:
Updated. [It looks like the `Initial`/`MaxRAMPercentage` is used when `-Xmx`/`-Xms` is not set](https://github.com/openjdk/jdk/blob/master/src/hotspot/share/runtime/arguments.cpp#L1588-L1590), so I'm setting the flags inside the `if !hasInitialRAMPercentage && !hasMaxRAMPercentage` branch.